clients/drcachesim/CMakeLists.txt

```diff
@@ -198,6 +198,7 @@ add_exported_library(drmemtrace_record_filter STATIC
   tools/filter/cache_filter.cpp
   tools/filter/type_filter.h
   tools/filter/encodings2regdeps_filter.h
+  tools/filter/func_marker_filter.h
```

Comment:
Rename to func_id_filter?

Could survey others to vote on it.

Reply:
Renamed to "func_id_filter.h" (and "func_id_filter_t" for the class name).